packages/kit/src/runtime/app/forms.js

```diff
@@ -161,14 +165,28 @@ export function enhance(form_element, submit = () => {}) {
 		let result;
 
 		try {
+			const is_valid_enctype = enctype?.match(
+				/application\/x-www-form-urlencoded|multipart\/form-data|text\/plain/
+			);
+
+			const headers = new Headers({
+				accept: 'application/json',
+				'Content-Type': is_valid_enctype ? enctype : 'application/x-www-form-urlencoded',
```

Comment:
@benmccann is it a breaking change if we start setting the `Content-Type` header for `use:enhance` POST requests where we previously didn't set this?

Reply:
@eltigerchino i modified this logic a bit in e22c88c478e247a0b3cf004e057961c034f358c5

We need to explicitly set this if we want to pass the value of `enctype` / `formenctype`